recipes/ios-run-test-groups-in-parallel.md

```diff
@@ -48,21 +48,21 @@ meta:
 pipelines:
   run_tests_groups:
```

Comment:
this is the same name as the stage

Reply:
renamed the pipeline to just `run_tests` (to avoid updating the screenshot)